src/main/java/org/broadinstitute/hellbender/tools/dataflow/transforms/bqsr/BaseRecalibratorFn.java

```diff
@@ -81,167 +58,126 @@
-                .and(new CountingReadFilter("Passes_Vendor_Quality_Check", ReadFilterLibrary.PASSES_VENDOR_QUALITY_CHECK));
-    }
-
-    private static GATKRead consolidateCigar(GATKRead read) {
-        // Always consolidate the cigar string into canonical form, collapsing zero-length / repeated cigar elements.
-        // Downstream code cannot necessarily handle non-consolidated cigar strings.
-        read.setCigar(AlignmentUtils.consolidateCigar(read.getCigar()));
-        return read;
-    }
-
-    private GATKRead resetOriginalBaseQualities(GATKRead read) {
-        if (!BRAC.useOriginalBaseQualities) {
-            return read;
-        }
-        return ReadUtils.resetOriginalBaseQualities(read);
-    }
-
-    private GATKRead setDefaultBaseQualities(GATKRead read) {
-        // if we are using default quals, check if we need them, and add if necessary.
-        // 1. we need if reads are lacking or have incomplete quality scores
-        // 2. we add if defaultBaseQualities has a positive value
-        if (BRAC.defaultBaseQualities < 0) {
-            return read;
-        }
-        byte reads[] = read.getBases();
-        byte quals[] = read.getBaseQualities();
-        if (quals == null || quals.length < reads.length) {
-            byte new_quals[] = new byte[reads.length];
-            Arrays.fill(new_quals, BRAC.defaultBaseQualities);
-            read.setBaseQualities(new_quals);
-        }
-        return read;
-    }
-
-    public RecalibrationTables getTables() {
-        return recalibrationEngine.getRecalibrationTables();
-    }
-
-    /**
-     * Returns the requested covariates, possibly mutated during the execution.
-     */
-    public StandardCovariateList getRequestedCovariates() {
-        return this.covariates;
-    }
-
-    /**
-     * @param rt recalibration tables
-     * @return the quantization information
-     */
-    public QuantizationInfo getQuantizationInfo(RecalibrationTables rt) {
-        return new QuantizationInfo(rt, BRAC.RAC.QUANTIZING_LEVELS);
-    }
-
-    private ReadTransformer makeReadTransform() {
-        ReadTransformer f0 = BaseRecalibratorWorker::consolidateCigar;
+        minimumQToUse = BRAC.PRESERVE_QSCORES_LESS_THAN;
 
-        ReadTransformer f =
-                f0.andThen(this::setDefaultBaseQualities)
-                        .andThen(this::resetOriginalBaseQualities)
-                        .andThen(ReadClipper::hardClipAdaptorSequence)
-                        .andThen(ReadClipper::hardClipSoftClippedBases);
 
-        return f;
     }
 
-
     /**
      * For each read at this locus get the various covariate values and increment that location in the map based on
      * whether or not the base matches the reference at this particular location.
      * The "knownLocations" list doesn't need to be complete but it must includes all those that overlap with the read.
+     *
+     * This will also remove any soft-clipped bases.
```

Comment:
Comment not accurate -- soft-clipped bases won't be removed (since this DoFn doesn't actually modify the reads), they just won't be considered in the error model.